src/common/util.h

```diff
@@ -61,6 +65,41 @@ inline int TVMPClose(FILE* stream) {
   return pclose(stream);
 #endif
 }
+
+/*
+ * gnulib sys_wait.h.in says on Windows
```

Comment:
To be on the safe side(license), I think we should avoid mention gnulib if you created the status handling code by yourself

Reply:
done